### DISCUSSION TITLE
[Idea]: Force some updates

### DISCUSSION THREAD

**Discussion by mtdvlpr** [sircharlo/meeting-media-manager#1732] [Ideas] [answered]
### Is your idea related to a problem?

No.

### Describe your idea

Sometimes it happens that there's a breaking/massive bug that is fixed in an update (like the bug that tried to remove a system folder). With the `disable auto updates` option, people might not get that update as soon as we would like. It might be useful to be able to force certain updates on the user (either ignore the disable option or a step further: forcefully update right away and restart).

### Describe alternatives you've considered

The alternative would be to announce it somehow and somewhere (in issues, discussions or release notes).

### Additional context

We could add a certain line in the release notes or title which we could read from the app to decide how to handle the available update from the auto-updater.

@sircharlo, what do you think?

**Comment by sircharlo** [comment, marked as answer]
Not a bad idea. We'd change "disable auto-updates" to "disable non-critical auto updates". It'd be reserved only for serious security vulnerabilities, or very bad bugs. (Hopefully, it would never be used!)

**Reply by mtdvlpr** [reply]
Alright, I guess a `[critical]` in the release title would be best/easiest. Then the question remains, which version do we implement:

When an available update title contains `[critical]`:
1. Immediately quit the app, update and restart
2. Update as soon as the application closes

Or, implement option 1 for `[critical]` and option 2 for `[important]`.
